java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/source/core/DbzCdcEventConsumer.java

```diff
@@ -76,6 +83,22 @@ public void handleBatch(
             DebeziumEngine.RecordCommitter<ChangeEvent<SourceRecord, SourceRecord>> committer)
             throws InterruptedException {
         var respBuilder = GetEventStreamResponse.newBuilder();
```

Comment:
Can `newBuilder` throw exception? If yes, will it cause dead lock because countDown is skipped.

Reply:
refactored